src/run.clj

```diff
@@ -0,0 +1,10 @@
+(ns run)
```

Comment:
I would put these under a namespace like `(ns clojure-camp-site.run)`

Reply:
@rafd Updated. I also had to update folder structure for this but I think it should be fine